src/common/include/SIRF/common/geometrical_info.h

```diff
@@ -0,0 +1,47 @@
+#ifndef SIRF_GEOMETRICAL_INFO_TYPE
+#define SIRF_GEOMETRICAL_INFO_TYPE
+
+const int DIM = 3;
```

Comment:
let's move it inside the class as a private static member

Reply:
Moved to template